packages/design-system/src/atoms/statusV2/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { View, StyleSheet } from 'react-native'
+import { useMemo } from 'react'
+import color from 'color'
+import { TypographyV2 } from '../typographyV2'
+import { useThemeV2 } from '../../hooks'
+
+const BORDER_RADIUS = 8
+type Status = 'warning' | 'error' | 'success' | 'action'
+
+export type StatusComponentProps = {
+  status: Status
+  title: string
+  textColor: string
+}
+
+const useTextColor = (status: Status) => {
+  const theme = useThemeV2()
+
+  switch (status) {
+    case 'warning':
+      return theme.base.warning
+    case 'error':
+      return theme.base.danger
+    case 'success':
+      return theme.base.success
+    case 'action':
+      return theme.base.highlight1
+    default:
+      return theme.text.black[100]
+  }
+}
+
+const useStyles = (status: Status) => {
+  const theme = useThemeV2()
+
+  const backgroundColor = useMemo(() => {
+    switch (status) {
+      case 'warning':
+        return theme.base.warning
+      case 'error':
+        return theme.base.danger
+      case 'success':
+        return theme.base.success
+      case 'action':
+        return theme.base.highlight1
+    }
+  }, [status, theme])
```

Comment:
I see a duplication here. Let's unify status-to-color functionality.

```suggestion
const useStatusToColor = (status: Status) => {
  const theme = useThemeV2()

  switch (status) {
    case 'warning':
      return theme.base.warning
    case 'error':
      return theme.base.danger
    case 'success':
      return theme.base.success
    case 'action':
      return theme.base.highlight1
    default:
      return theme.text.black[100]
  }
}

const useStyles = (status: Status) => {
  const theme = useThemeV2()

  const backgroundColor = useStatusToColor(status)
```